Testing/SystemTests/tests/analysis/AlignAndFocusPowderFromFilesTest.py

```diff
@@ -7,15 +7,10 @@
 #pylint: disable=no-init,invalid-name,attribute-defined-outside-init
 import systemtesting
 from mantid.simpleapi import *
+from mantid.api import WorkspaceFactory
+import numpy as np
 import os
-
-
-def _skip_test():
-    """Helper function to determine if we run the test"""
-    import platform
-
-    # Only runs on RHEL6 at the moment
-    return "Linux" not in platform.platform()
+import timing
```

Comment:
I got `no module found` for `timing`. Add as dependency or switch to different module?